lib/schemas.js

```diff
@@ -78,4 +78,8 @@ Schemas.prototype.getSchemaAnyway = function (schema) {
   return schema
 }
 
+Schemas.prototype.getAllSchemas = function () {
+  return this.store
```

Comment:
I think this should return a copy.

Reply:
Ok. Will `JSON.parse(JSON.stringify())` fit like it's done in fastify-swagger, or it's better to add module for this, or just some helper function?

Reply:
I think it’s ok doing an Object.assign(), a shallow copy is enough.

Reply:
@mcollina If it will be a shallow copy, then there will be possibility to change something in schemas in runtime (e.g. properties could be deleted or something like that). And in this way I see no reason of returning shallow copy instead of a reference to original object.

Reply:
The way `addSchema` has been added is to mask the internal implementation. I'm not convinced we are storing these in the best possible way. That's why I prefer a shallow copy.